openBCISample.js

```diff
@@ -3,63 +3,70 @@ var gaussian = require('gaussian');
 var outliers = require('outliers');
 var stats = require('scientific-statistics');
 
+var k = require('./openBCIConstants');
+
 /** Constants for interpreting the EEG data */
 // Reference voltage for ADC in ADS1299.
 //   Set by its hardware.
 const ADS1299_VREF = 4.5;
-// Assumed gain setting for ADS1299.
-//   Set by its Arduino code.
-const ADS1299_GAIN = 24.0;
 // Scale factor for aux data
 const SCALE_FACTOR_ACCEL = 0.002 / Math.pow(2,4);
-// Scale factor for channelData
-const SCALE_FACTOR_CHANNEL = ADS1299_VREF / ADS1299_GAIN / (Math.pow(2,23) - 1);
+// X, Y, Z
+const ACCEL_NUMBER_AXIS = 3;
+// Default ADS1299 gains array
 
+// For computing Goertzel Algorithm
+// See: http://www.embedded.com/design/configurable-systems/4024443/The-Goertzel-Algorithm
+// This is N
```

Comment:
N is what `GOERTZEL_BLOCK_SIZE` is referred to as in the referenced tutorial provided in the link above. Fixed the comment for this